installer_scripts/install-yb-voyager

```diff
@@ -141,18 +153,26 @@ update_yb_voyager_bashrc() {
 
 
 install_yb_voyager() {
-	output "Installing yb-voyager."
-	inside_repo=`git rev-parse --is-inside-work-tree 2> /dev/null || echo "false"`
-	if [ ${inside_repo} == "true" ]
+	output "Installing yb-voyager:${VERSION}."
+	if [ "${VERSION}" == "latest" ]
 	then
-		pushd `git rev-parse --show-toplevel` > /dev/null
-		pushd yb-voyager > /dev/null
-		$GO install
-		popd > /dev/null
-		popd > /dev/null
-	else
 		$GO install github.com/yugabyte/yb-voyager/yb-voyager@latest
+		sudo mv -f $HOME/go/bin/yb-voyager /usr/local/bin
+		return
+	fi
+
+	inside_repo=`git rev-parse --is-inside-work-tree 2> /dev/null || echo "false"`
```

Comment:
nit: adding a else condition or comment about "installing from local" would help in code readability.